modules/library/metadata/src/main/java/org/geotools/factory/GeoTools.java

```diff
@@ -196,6 +196,22 @@ private static Properties loadProperites(String resource) {
     /**
      * The {@linkplain System#getProperty(String) system property} key for the default
      * value to be assigned to the {@link Hints#
+     * ENTITY_RESOLVER} hint.
+     *
+     * This setting specifies the XML Entity resolver to be used when configuring a SAXParser,
+     * by default it NoLocalExternalEntityResolver.
+     *
+     * @see Hints#ENTITY_RESOLVER
+     * @see #getDefaultHints
+     */
+    public static final String ENTITY_RESOLVER = "org.xml.sax.EntityResolver";
```

Comment:
We are not doing that yet, we are currently unsafe - just to confirm we can supply PreventLocalEntityResolver via a hint.